### DISCUSSION TITLE
debugging sc-im code with vscode

### DISCUSSION THREAD

**Discussion by CesiraBuffarlotti** [andmarti1424/sc-im#895] [General]
I don't know anything about C/C++ but nevertheless I wanted to try debugging the sc-im code because I would like to fix the columns to a width more congenial to me: the size of “Current format is 4 2 0” obtained by using “f<” several times. For the record :autofit and aa do not work for me.
That said, when I run or debug main.c I immediately get a notable set of errors.
```
--Executing task: C/C++: gcc build active file. 

Starting build...
/usr/bin/gcc -fdiagnostics-color=always -g /Projects/sc-im/src/main.c -o /Projects/sc-im/src/main
/usr/bin/ld: /tmp/ccZQofqW.o: warning: relocation against `inputline' in read-only section `.text'
/usr/bin/ld: /tmp/ccZQofqW.o: in function `main':
/Projects/sc-im/src/main.c:186: undefined reference to `create_dictionary'
/usr/bin/ld: /Projects/sc-im/src/main.c:187: undefined reference to `store_default_config_values'
/usr/bin/ld: /Projects/sc-im/src/main.c:193: undefined reference to `get_conf_int'...
```
How can I create a working task.json?

**Comment by andmarti1424** [comment]
Couldnt you just format the columns?

**Comment by CesiraBuffarlotti** [comment]
I can do it but this implies the creation of two files. One for me, formatted according to my desiderata, and another for my colleagues who read it on the cloud
